src/components/Toolbar/__tests__/Toolbar.test.jsx

```diff
@@ -55,4 +56,22 @@ describe('Toolbar Component Tests', function() {
     expect(queryByText('Partners')).toBeInTheDocument();
     expect(queryByText('Sign Up')).not.toBeInTheDocument();
   });
+
+  test('Toolbar resized to width over medium screen breakpoint; Mobile Navigation Menu Closed', () => {
+    const { queryByLabelText, getByTestId } = setup();
+    window.innerWidth = MEDIUM_BREAKPOINT - 10;
+    window.dispatchEvent(new Event('resize'));
+    queryByLabelText('hamburger')
```

Comment:
I am confused to why this line is here, I don't think it does anything.

Reply:
good catch, shall remove